client/asset/btc/rpcclient.go

```diff
@@ -240,19 +241,24 @@ func (wc *rpcClient) callHashGetter(method string, args anylist) (*chainhash.Has
 
 // getBlock fetches the MsgBlock.
 func (wc *rpcClient) getBlock(h chainhash.Hash) (*wire.MsgBlock, error) {
-	var txB dex.Bytes
+	var blkB dex.Bytes
 	args := anylist{h.String()}
 	if wc.booleanGetBlock {
 		args = append(args, false)
 	} else {
 		args = append(args, 0)
 	}
-	err := wc.call(methodGetBlock, args, &txB)
+	err := wc.call(methodGetBlock, args, &blkB)
 	if err != nil {
 		return nil, err
 	}
+
+	if wc.blockDeserializer != nil {
+		return wc.blockDeserializer(blkB)
+	}
+
```

Comment:
Can we move this logic to before the call to `newRPCClient`? 

That's the pattern [I settled on](https://github.com/decred/dcrdex/blob/e99961014847f355490cf864d30b0879a8c5ff90/client/asset/btc/btc.go#L766-L769) for #1570, with a [`deserializeBlock` helper function](https://github.com/decred/dcrdex/blob/e99961014847f355490cf864d30b0879a8c5ff90/client/asset/btc/btc.go#L3778).

Reply:
Sure thing.  I hadn't noticed that, but I makes sense there